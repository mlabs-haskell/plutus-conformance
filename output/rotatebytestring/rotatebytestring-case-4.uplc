[ [ (builtin rotateByteString) (con bytestring #ebfc) ] (con integer -5) ]