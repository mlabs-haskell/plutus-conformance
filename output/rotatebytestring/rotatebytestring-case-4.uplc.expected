(con bytestring #e75f)